recipes/kapteyn/meta.yaml

```diff
@@ -0,0 +1,105 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "kapteyn" %}
+{% set version = "2.3" %}
+{% set sha256 = "87ab1809880ea6ede12bb41e675d76d66264f6e66619e1727c65e36254a4d9af" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    - cython
+    - numpy
```

Comment:
Please change from `numpy` to `numpy x.x` here and in the `run` section since this package appears to depend on the numpy C-API. See the "Requirements" section of:

https://www.continuum.io/blog/developer-blog/whats-old-and-new-conda-build

and 

http://conda.pydata.org/docs/building/meta-yaml.html#run